packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -849,12 +830,6 @@ export class StorageService extends EventEmitter {
       this.logger('Certificate is either null or undefined, not saving to db')
       return false
     }
-    const verification = await verifyUserCert(payload.rootPermsData.certificate, payload.certificate)
```

Comment:
Why did you remove that?

Reply:
I'd recommend adding a access controller for that verifies certificate validity. This sort of validation needs to happen on each peer when receiving (replicating) data.

Reply:
@EmiM Because it doesn't make sense to verify that on write, only owner adds certificates and he trusts himself. 
@leblowl I can do that, but keep in mind that it will break the current community, adding access controller to existing db means creating new database and losing access to the old set of certificates - should I do that now? in other words - after releasing our community will break

Reply:
@leblowl I think this needs a separate discussion on how to refactor storage module (it was planned for a very long time already), and how to actually design it to make it stronger in terms of both backward compatibility and upgrades. I can share my thoughts on that. I learned that the hard way by being unaware that adding access controllers means creating new db and I've dropped all the channels in community.

Reply:
@vinkabuki I'm okay with implementing this in a separate PR. We should make an issue.

Reply:
https://github.com/TryQuiet/quiet/issues/1893
https://github.com/TryQuiet/quiet/issues/1899